code/modules/mob/living/life.dm

```diff
@@ -91,20 +85,8 @@
 //This updates the health and status of the mob (conscious, unconscious, dead)
 /mob/living/proc/handle_regular_status_updates()
 
-	updatehealth()
-
-	if(stat != DEAD)
-
-		if(paralysis)
-			stat = UNCONSCIOUS
-
-		else if(status_flags & FAKEDEATH)
-			stat = UNCONSCIOUS
-
-		else
-			stat = CONSCIOUS
-
-		return 1
+	updatehealth("handle regular status updates")
```

Comment:
I also wonder if we should be calling this every `Life` as well, again, for the same reason that it's bad to call `updatecanmove` every `Life`.

Reply:
I'll excise those updates and see what happens